submit/submit.go

```diff
@@ -23,9 +23,10 @@ import (
 )
 
 type cliargs struct {
-	Image     string   `arg:"-i,required,help:ECR image like $acct.dkr.ecr.$region.amazonaws.com/$tag"`
+	Image     string   `arg:"-i,required,help:image like $acct.dkr.ecr.$region.amazonaws.com/$image:$tag or $image:$tag"`
+	Registry  string   `arg:"env" help:"Docker image registry, defaults to $acct.dkr.ecr.$region.amazonaws.com"`
```

Comment:
Can you fix this so the help will still show

Reply:
Working on it. Finicky syntax! BTW I notice that `-r` is the short flag for both Retries and Role. Not sure what happens if you use `-r` in a command line. ;-)

Reply:
Actually it seems like there are a number of arguments for which the help does not show.

Reply:
So the help does show. But not if you pipe the output through grep, because the help is not always on the same line as the flags. For example:


```
Options:
  --image IMAGE, -i IMAGE
                         image like $acct.dkr.ecr.$region.amazonaws.com/$image:$tag or $image:$tag
```

I *am* going to send you another commit though, because I made the help text for REGISTRY more consistent with other help tags that have default values.